### PR TITLE
Support v6 of the Unifi controller

### PR DESCRIPTION
**Pull request by paultyng**

### CODE REVIEW

**Comment by paultyng** [comment]
Started a branch for testing v5 and v6 of the controller, but need to address the v6 bugs.

**Comment by paultyng** [comment]
@kurtmc this is my initial attempt at supporting both v5 and v6 (and testing them both), let me know your thoughts!

File: controller.sh [hunk 1]
```diff
@@ -39,13 +41,13 @@ case "$1" in
     UNIFI_API="https://localhost:${DOCKER_HTTPS_PORT}/" \
     UNIFI_ACC_WLAN_CONCURRENCY="4" \
     UNIFI_INSECURE="true" \
-    go test -v -cover -count 1 ./internal/provider
+    go test -v -cover -count 1 -run 'WLAN|AP' ./internal/provider
```
Comment (kurtmc) [inline controller.sh:44]: What does this do?

**Review by kurtmc — APPROVED** [review]
Looks really good to me. Ran it on my laptop without too much issue. For me `./controller.sh reset` doesn't quite work because the files created by the docker container is owned by root.

File: controller.sh (hunk 1 above)
Comment (paultyng) [reply controller.sh:44]: Oops accidentally left that in, I was just contraining the tests to WLAN/AP tests, need to rerun this with all tests.

**Pull request merged**